src/environment_setup/conda.ml

```diff
@@ -35,114 +63,49 @@ let installed ~(run_program : Machine.Make_fun.t) ~host ~install_path =
           `Internet_access; `Self_identification ["conda"; "installation"]
         ]
         Program.(
-          exec ["mkdir"; "-p"; install_path]
-          && exec ["cd"; install_path]
+          exec ["mkdir"; "-p"; install_dir]
+          && exec ["cd"; conda_env.install_path]
           && Workflow_utilities.Download.wget_program url
           && shf "bash Miniconda3-latest-Linux-x86_64.sh -b -p %s" install_dir))
```

Comment:
damn you Miniconda installation procedure. OK - fixed it with the latest commit.